examples/visualization/plot_meg_sensors.py

```diff
@@ -14,23 +14,24 @@
 
 from mayavi import mlab
 
-import mne
 from mne.io import Raw, read_raw_ctf, read_raw_bti, read_raw_kit
 from mne.datasets import sample, spm_face
 from mne.viz import plot_trans
 
 print(__doc__)
 
-bti_path = op.abspath(op.dirname(mne.__file__)) + '/io/bti/tests/data/'
-kit_path = op.abspath(op.dirname(mne.__file__)) + '/io/kit/tests/data/'
+bti_path = op.join(op.abspath(op.dirname(__file__)), '..', '..', 'mne', 'io',
+                   'bti', 'tests', 'data')
+kit_path = op.join(op.abspath(op.dirname(__file__)), '..', '..', 'mne', 'io',
+                   'kit', 'tests', 'data')
```

Comment:
Same comment as @agramfort made before about these paths -- we shouldn't assume that the `example` file is still in its original location, so `mne.__file__` shoudl be more robust. What did you end up doing in your last PR to deal with this comment and get things working?